### ISSUE TITLE
Inconsistent validity state SERVER_SHUTDOWN

### ISSUE THREAD

**Issue by Askaholic** [micheljung/faf-java-server#110]
This validity state is not reflected in the db migrations, or in the python server or java api.

**Comment by micheljung** [comment]
True, I didn't add it yet because if I did, next thing I'd be asked is to add it to the python server as well, and when I added it to the python server someone would ask why I added it w/o implementing it etc.

Also, it might not be the last additional validity state and I don't feel like updating 4 repos every time I add one, as long as this server is WIP.

So I'm waiting, because once the java server goes into production the python server will be deprecated and then there is no need to update it.

But it surely must not be forgotten.
